debian/control

```diff
@@ -0,0 +1,39 @@
+Source: libcontentfeed
+Priority: standard
+Maintainer: Sam Spilsbury <sam@endlessm.com>
+Build-Depends: debhelper (>= 9),
+               dh-autoreconf,
+               gobject-introspection,
+               libeos-shard-0-dev,
+               libgirepository1.0-dev,
+               libglib2.0-bin,
+               libglib2.0-dev,
+               libsoup2.4-dev,
+               pkg-config
+Standards-Version: 3.9.4
+Section: non-free/libs
```

Comment:
It's not non-free

Reply:
True. I put it into `utils` since I can't think of a better section name to put it in :laughing: